src/renderers/shaders/UniformsLib.js

```diff
@@ -18,6 +18,7 @@ var UniformsLib = {
 		offsetRepeat: { value: new Vector4( 0, 0, 1, 1 ) },
 
 		specularMap: { value: null },
+		uSpecularStrength: { value : 1 },
```

Comment:
I have expressed my opinion on this topic in my first comment above, and more recently in https://github.com/mrdoob/three.js/issues/9339#issuecomment-312527206.

It appears @bhouston expressed his opinion is his first comment above, too.

So this can be closed, and we can address @pailhead's interest via #9339 .